src/components/InjectedComponents/PostDialog.tsx

```diff
@@ -301,20 +303,20 @@ export function PostDialog(props: PostDialogProps) {
                         metadata.val.token &&
                         isDAI(metadata.val.token)
                     activeUI.taskPasteIntoPostBox(
-                        geti18nString('additional_post_box__steganography_post_pre', String(Date.now())),
+                        t('additional_post_box__steganography_post_pre', { random: String(Date.now()) }),
                         {
-                            warningText: geti18nString('additional_post_box__encrypted_failed'),
+                            warningText: t('additional_post_box__encrypted_failed'),
                             shouldOpenPostDialog: false,
                         },
                     )
                     activeUI.taskUploadToPostBox(encrypted, {
                         template: isEth ? 'eth' : isDai ? 'dai' : 'default',
-                        warningText: geti18nString('additional_post_box__steganography_post_failed'),
+                        warningText: t('additional_post_box__steganography_post_failed'),
                     })
                 } else {
-                    let text = geti18nString('additional_post_box__encrypted_post_pre', encrypted)
+                    let text = t('additional_post_box__encrypted_post_pre', { encrypted })
                     if (metadata.ok) {
-                        if (getCurrentLanguage() === 'zh') {
+                        if (i18n.language.includes('zh')) {
                             text =
```

Comment:
The text write into messages file?

Reply:
It's an ad-hoc code. will remove in future so not going to add it.